server/lib/s3backup/__init__.py

```diff
@@ -154,7 +154,7 @@ def put_tarball(self, Name=None, Body=None, Size=0, ContentMD5=None, Bucket=None
                     })
             except ClientError as e:
                 self.logger.error(
```

Comment:
Here you could use self.logger.exception to print out the stack trace.

Reply:
We are actively trying to avoid stack traces in the log messages when handling "known" error conditions.  This is why we just print the nature of the exception in the error message.  In other places where we don't expect failures, we dump the whole exception (e.g. line 133).